src/cores/deepin.yaml

```diff
@@ -0,0 +1,9 @@
+include: [
+  "./common.yaml",
+  "../packages/deepin.yaml"
```

Comment:
took gnome file as base , likely would need edits later on - but good for start point